src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="submit"
+      class="form"
+    >
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="form__input form__input_big"
```

Comment:
The `alt` attribute is missing in your input tags. Although it's not mandatory, it's a good practice to include it as it improves accessibility. The `alt` attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the `src` attribute, or if the user uses a screen reader).